### ISSUE TITLE
CUDA version

### ISSUE THREAD

**Issue by alexfikl** [NatsuYukiHaruhana/Fractal-Dimension-Computation#2]
This was supposed to have a CUDA version too?

**Comment by NatsuYukiHaruhana** [comment]
Yep!~ For initial testing, it was removed so that I could run it on a laptop without an nVidia GPU.

**Comment by alexfikl** [comment]
At least with cmake you can add a flag `-DUSE_CUDA` (or something) to enable+compile the GPU code as well. For a quick example:
https://github.com/lukeyeager/cmake-cuda-example/blob/master/CMakeLists.txt

I imagine there's some way to set up build profiles for Visual Studio / VS Code / whatever, so that would be nice to have :grin:

**Comment by alexfikl** [comment]
There seems to be a CUDA version in here now. Any chance we could include it in the `CMakeLists.txt` file to make it easier to compile? :grin:
